### ISSUE TITLE
:IcedCleanNs doesn't add a newline after :require

### ISSUE THREAD

**Issue by noahmoss** [liquidz/vim-iced#453]
Hey @liquidz! I've noticed that `:IcedCleanNs` won't add a newline after the `:require` form if the namespace is otherwise correct. However, it will if there's anything that needs to be cleaned. Is this the intended behavior? If so is there any other command that can be used to add the newline to the ns form?

For example, in this namespace:

```
(ns test
  (:require [clojure.string :as str]))

(str/replace "foo" "o" "a")
```

running `:IcedCleanNs` will not make any changes. However, in this ns:
```
(ns test
  (:require [clojure.string :as str]
            [clojure.java.io :as io]))

(str/replace "foo" "o" "a")
```
running `:IcedCleanNs` will result in 
```
(ns test
  (:require
   [clojure.string :as str]))

(str/replace "foo" "o" "a")
```

**Comment by liquidz** [comment]
@noahmoss Thanks for your reporting!

The behavior inserting new line after `:require` is due to the following option. 
(This option is a [refactor-nrepl's option](https://github.com/clojure-emacs/refactor-nrepl#configuration))
https://liquidz.github.io/vim-iced/vim-iced.html#g%3Aiced%23refactor%23insert_newline_after_require

And when refactor-nrepl responses that there are no changes, vim-iced does not update `ns` form, so the newline is not added.

>  If so is there any other command that can be used to add the newline to the ns form?

No commands for now, but  the behavior is not unified, so I'll find some solution.

**Comment by noahmoss** [comment]
@liquidz That makes sense. In my case I am trying to update a large number of existing namespaces to use the formatting with the newline, so having the behavior be consistent would be great if possible. Thanks for taking a look!

**Comment by liquidz** [comment]
@noahmoss I tried to fix the behavior.
Could you try [next_release](https://github.com/liquidz/vim-iced/tree/next_release) branch?

**Comment by noahmoss** [comment]
Seems to work, thanks!

**Comment by liquidz** [comment]
Thanks for your confirmation!
Just released v3.14.3196
